plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -513,23 +513,23 @@ private static RowType toRowType(List<ColumnMetadata> columns)
                             .put("t_float", createDoubleColumnStatistics(OptionalDouble.of(123.25), OptionalDouble.of(567.58), OptionalLong.of(9), OptionalLong.of(10)))
                             .put("t_string", createStringColumnStatistics(OptionalLong.of(10), OptionalLong.of(50), OptionalLong.of(3), OptionalLong.of(7)))
                             .put("t_varchar", createStringColumnStatistics(OptionalLong.of(100), OptionalLong.of(230), OptionalLong.of(5), OptionalLong.of(3)))
-                            .put("t_char", createStringColumnStatistics(OptionalLong.of(5), OptionalLong.of(500), OptionalLong.of(1), OptionalLong.of(4)))
-                            .put("t_varbinary", createBinaryColumnStatistics(OptionalLong.of(4), OptionalLong.of(300), OptionalLong.of(1)))
+                            .put("t_char", createStringColumnStatistics(OptionalLong.of(5), OptionalLong.of(50), OptionalLong.of(1), OptionalLong.of(4)))
```

Comment:
Why this change? This indicates to me that are we somehow calculating data size differently for Thrift vs Glue?

Reply:
Here I'm using the `ThriftMetastoreUtil#getAverageColumnLength` function I had to change the values because Glue  reject logically inconsistent statistics.
In particular:

**t_varbinary**
```
test data: 
 rows:  15
 nulls: 1
 max_size:   4
 total_size: 300

avg size: 300/(15rows - 1null) = 20...  > 4 (max)  # ERROR 
```

**t_char**
```
 rows: 15
 nulls: 1 
 max_size: 5
 total_size: 500

avg size: 500/(15rows - 1null) = 35... > 5 (max) # ERROR 
```

Reply:
> because Glue reject logically inconsistent statistics

Are we sure that statistics computation mechanism used by Tino will not trigger this safety valve?
cc: @findepi